spec/models/user_spec.rb

```diff
@@ -63,6 +63,9 @@
 
   describe 'scopes' do
     context 'role scopes' do
+      roles_except_organizer = User::ROLES
```

Comment:
I'm undecided whether it's good to have the test run use a MP pattern as well. I feel that it would be better for this test to catch regressions by defining the list of roles as a static list, not re-using `User::ROLES`